runtime/src/juvix/io.c

```diff
@@ -32,6 +32,18 @@ void io_flush() {
     }
 }
 
+static void io_print(word_t x) {
```

Comment:
It's odd these changes appear in the diff, is this a GitHub bug? We just merged these into main in https://github.com/anoma/juvix/pull/1969 ?

Reply:
Apparently, GitHub believes that the commit for #1969 is a part of this PR. Strange. There is no option to rebase. I don't know what to do with this.